2021-04-22 19:44:29,368 INFO    MainThread:6610 [wandb_setup.py:_flush():69] setting env: {}
2021-04-22 19:44:29,368 INFO    MainThread:6610 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-22 19:44:29,368 INFO    MainThread:6610 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '7fc5bf615755bb4190c75b650ad7709a896352f4'}
2021-04-22 19:44:29,368 INFO    MainThread:6610 [wandb_init.py:_log_setup():336] Logging user logs to /home/wilfred/Downloads/github/Python_Projects/videoPrediction/wandb/run-20210422_194429-227sq6da/logs/debug.log
2021-04-22 19:44:29,369 INFO    MainThread:6610 [wandb_init.py:_log_setup():337] Logging internal logs to /home/wilfred/Downloads/github/Python_Projects/videoPrediction/wandb/run-20210422_194429-227sq6da/logs/debug-internal.log
2021-04-22 19:44:29,369 INFO    MainThread:6610 [wandb_init.py:_jupyter_setup():287] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f6974414128>
2021-04-22 19:44:29,370 INFO    MainThread:6610 [wandb_init.py:init():369] calling init triggers
2021-04-22 19:44:29,370 INFO    MainThread:6610 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {'num_epochs': 100, 'batch_size': 32, 'height': 96, 'width': 96}
2021-04-22 19:44:29,370 INFO    MainThread:6610 [wandb_init.py:init():391] re-initializing run, found existing run on stack: 88yzxle7
2021-04-22 19:44:29,372 INFO    MainThread:6610 [wandb_run.py:finish():1167] finishing run wilfredkisku/videoPrediction/88yzxle7
2021-04-22 19:44:29,373 INFO    MainThread:6610 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 0
2021-04-22 19:44:29,373 INFO    MainThread:6610 [wandb_run.py:_restore():1466] restore
2021-04-22 19:44:29,834 INFO    MainThread:6610 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 796
  total_bytes: 796
}

2021-04-22 19:44:30,241 INFO    MainThread:6610 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 796
  total_bytes: 6935
}

2021-04-22 19:44:31,017 INFO    MainThread:6610 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6937
  total_bytes: 6937
}

2021-04-22 19:44:31,125 INFO    MainThread:6610 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6937
  total_bytes: 6937
}

2021-04-22 19:44:31,228 INFO    MainThread:6610 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6937
  total_bytes: 6937
}

2021-04-22 19:44:31,335 INFO    MainThread:6610 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6937
  total_bytes: 6937
}

2021-04-22 19:44:31,439 INFO    MainThread:6610 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6937
  total_bytes: 6937
}

2021-04-22 19:44:31,542 INFO    MainThread:6610 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6937
  total_bytes: 6937
}

2021-04-22 19:44:31,646 INFO    MainThread:6610 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6937
  total_bytes: 6937
}

2021-04-22 19:44:32,932 INFO    MainThread:6610 [wandb_run.py:_show_files():1838] logging synced files
2021-04-22 19:44:32,939 INFO    MainThread:6610 [wandb_init.py:init():418] starting backend
2021-04-22 19:44:32,939 INFO    MainThread:6610 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-22 19:44:32,944 INFO    MainThread:6610 [backend.py:ensure_launched():123] starting backend process...
2021-04-22 19:44:32,951 INFO    MainThread:6610 [backend.py:ensure_launched():128] started backend process with pid: 7151
2021-04-22 19:44:32,952 INFO    MainThread:6610 [wandb_init.py:init():423] backend started and connected
2021-04-22 19:44:32,997 INFO    MainThread:6610 [wandb_init.py:init():465] updated telemetry
2021-04-22 19:44:33,000 INFO    MainThread:6610 [wandb_init.py:init():484] communicating current version
2021-04-22 19:44:34,155 INFO    MainThread:6610 [wandb_init.py:init():489] got version response 
2021-04-22 19:44:34,156 INFO    MainThread:6610 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-22 19:44:34,710 INFO    MainThread:6610 [wandb_init.py:init():522] starting run threads in backend
2021-04-22 19:44:36,010 INFO    MainThread:6610 [wandb_run.py:_console_start():1524] atexit reg
2021-04-22 19:44:36,011 INFO    MainThread:6610 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.WRAP
2021-04-22 19:44:36,011 INFO    MainThread:6610 [wandb_run.py:_redirect():1414] Wrapping output streams.
2021-04-22 19:44:36,011 INFO    MainThread:6610 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-22 19:44:36,011 INFO    MainThread:6610 [wandb_init.py:init():546] run started, returning control to user process
2021-04-22 19:44:36,358 INFO    MainThread:6610 [wandb_init.py:_pause_backend():249] pausing backend
2021-04-22 19:44:36,358 INFO    MainThread:6610 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-04-22 19:50:31,700 INFO    MainThread:6610 [wandb_init.py:_resume_backend():258] resuming backend
2021-04-22 19:50:31,725 INFO    MainThread:6610 [wandb_setup.py:_flush():69] setting env: {}
2021-04-22 19:50:31,725 INFO    MainThread:6610 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-22 19:50:31,725 INFO    MainThread:6610 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '7fc5bf615755bb4190c75b650ad7709a896352f4'}
2021-04-22 19:50:31,725 INFO    MainThread:6610 [wandb_init.py:_log_setup():336] Logging user logs to /home/wilfred/Downloads/github/Python_Projects/videoPrediction/wandb/run-20210422_195031-2w6ec3qq/logs/debug.log
2021-04-22 19:50:31,725 INFO    MainThread:6610 [wandb_init.py:_log_setup():337] Logging internal logs to /home/wilfred/Downloads/github/Python_Projects/videoPrediction/wandb/run-20210422_195031-2w6ec3qq/logs/debug-internal.log
2021-04-22 19:50:31,726 INFO    MainThread:6610 [wandb_init.py:init():369] calling init triggers
2021-04-22 19:50:31,727 INFO    MainThread:6610 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {'num_epochs': 100, 'batch_size': 32, 'height': 96, 'width': 96}
2021-04-22 19:50:31,727 INFO    MainThread:6610 [wandb_init.py:init():391] re-initializing run, found existing run on stack: 227sq6da
2021-04-22 19:50:31,729 INFO    MainThread:6610 [wandb_run.py:finish():1167] finishing run wilfredkisku/videoPrediction/227sq6da
